presto-main/src/main/java/io/prestosql/sql/planner/PlanOptimizers.java

```diff
@@ -460,6 +466,7 @@ public PlanOptimizers(
                                 new ImplementIntersectAsUnion(metadata),
                                 new ImplementExceptAsUnion(metadata))),
                 new LimitPushDown(), // Run the LimitPushDown after flattening set operators to make it easier to do the set flattening
+                simplifyCountOverConstantOptimizer, // Run SimplifyCountOverConstant before column pruning to ensure the aggregation pushdown to connectors.
```

Comment:
Instead of adding new `IterativeOptimizer`, maybe you could add this rule to existing one. Maybe here: https://github.com/prestosql/presto/pull/4473/files#diff-ac936e954fdd13abb49900e706ea17fcL391?

Reply:
@kokosing  The link doesn't seem to work for me.

Reply:
It works, but you don't see it. It marks the line that is hidden.

![image](https://user-images.githubusercontent.com/7569403/87775631-b76e6800-c826-11ea-989c-eaef5e41df9b.png)

Reply:
Thanks for (not) noticing this and letting me know.